src/app/zap-templates/zcl/zcl.json

```diff
@@ -635,6 +636,7 @@
         "Power Topology": ["FeatureMap"],
         "Valve Configuration and Control": ["RemainingDuration"],
         "Boolean State Configuration": ["CurrentSensitivityLevel"],
+        "Water Heater Management": ["FeatureMap"],
```

Comment:
Again, this bit should come with the implementation.

Reply:
Would you like to see this line removed until the implementation is submitted for review?

Reply:
Yes, see https://github.com/project-chip/connectedhomeip/pull/33673#discussion_r1654702141

Reply:
Lines removed